packages/dd-trace/src/noop/proxy.js

```diff
@@ -75,6 +75,10 @@ class Tracer {
     this.appsec.setUser(user)
     return this
   }
+
+  get TracerProvider () {
+    return require('../opentelemetry/tracer_provider')
+  }
```

Comment:
Extra bug fix to allow accessing `TracerProvider` _before_ doing `tracer.init(...)`.